src/debug-lines.ts

```diff
@@ -156,8 +156,8 @@ class DebugLines {
                 vertex_position: SEMANTIC_POSITION,
                 vertex_color: SEMANTIC_COLOR
             },
-            vertexCode: vshader,
-            fragmentCode: fshader
+            vertexGLSL: vshader,
```

Comment:
at some point we should covert these to WGSL as well, and remove WGSL wasm transpilers completely

Reply:
(unless needed for other shaders)